packages/debug/src/browser/debug-session-manager.ts

```diff
@@ -105,11 +107,23 @@ export class DebugSessionManager {
     @inject(MessageService)
     protected readonly messageService: MessageService;
 
+    @inject(ContextKeyService)
+    protected readonly contextKeyService: ContextKeyService;
+
+    protected debugTypeKey: ContextKey<string>;
+    protected inDebugModeKey: ContextKey<boolean>;
+
     @postConstruct()
     protected init(): void {
+        this.debugTypeKey = this.contextKeyService.createKey<string>('debugType', undefined);
```

Comment:
couldn't find this context in https://github.com/Microsoft/vscode-docs/blob/master/docs/getstarted/keybindings.md#when-clause-contexts